src/wp-includes/class-wp-term-query.php

```diff
@@ -730,6 +730,8 @@ public function get_terms() {
 		 */
 		$clauses = apply_filters( 'terms_clauses', compact( $pieces ), $taxonomies, $args );
 
+		$fields_is_filtered = $fields !== $clauses['fields'];
```

Comment:
`$fields` is set with the result of the `get_terms_fields` filter so it's possible this will show as `false` even though the value has been modified.

I think the unfiltered value is `implode( ', ', $selects )` but a logic check on that statement is needed :)

Reply:
Good catch @peterwilsoncc . I have actioned your feedback. https://github.com/WordPress/wordpress-develop/pull/4363/commits/afeb2d00d48fff5332ec202f1e7bfbd5308a29c6